### PR TITLE
Feature/set up api sa

### PR DESCRIPTION
**Pull request by StevieAmb**
What (if any) features are you implementing?
- The feature of pulling the different data from the api based on api calls

What (if anything) did you refactor?
- n/a

Were there any issues that arose?
- Figuring out how to make things work with typescript and how to declare functions inside of the interface. Had to declare the types of parameters, in order for it to work.

Is there anything that you need from your teammate?
- You all are AWESOME. Your support is amazing.

Any other comments, questions, or concerns?
- No

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -54,12 +56,30 @@ class App extends Component<{}, State> {
...
+   
+  }
+
+  getCategory = (category: string) => {
+    getArtDetails(category)
+    .then(data => {
+      console.log(data.data)
+      this.setState({artPieces: data.data})
+    })
+  }
```
Comment (mr-ryan12) [inline src/App.tsx:69]: Excellent work successfully fetching the data!

File: src/App.tsx [hunk 2]
```diff
@@ -54,12 +56,30 @@ class App extends Component<{}, State> {
...
+
+  // getImage = () => {
+    
+
+  //   })
+  // }
```
Comment (mr-ryan12) [inline src/App.tsx:75]: We could probably remove this commented out code.

File: src/apiCalls.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  return fetch(`https://api.artic.edu/api/v1/artworks/search?q=${category}&fields=image_id,title,date_start,date_end`)
+  .then(response => response.json())
+}
+
+const getArtImage = (id: string) => {
+  return fetch(`https://www.artic.edu/iiif/2/${id}/full/843,/0/default.jpg`)
+  .then(response => response.json())
+}
+
+export { getArtDetails, getArtImage }
```
Comment (mr-ryan12) [inline src/apiCalls.tsx:11]: Really like we found out useful functionality of the API! Really simplifies the process of receiving data!

File: src/CategoryCard/CategoryCard.tsx [hunk 4]
```diff
@@ -2,12 +2,13 @@ import React from 'react'
...
 interface Props {
+  acquireCategory: (text: string) => void
   category: string;
 }
 
 const CategoryCard: React.FC<Props> = props => {
   return (
-    <section className="category-card">
+    <section className="category-card" onClick={() => props.acquireCategory(props.category)}>
       {/* <img src="https://www.artic.edu/iiif/2/1adf2696-8489-499b-cad2-821d7fde4b33/full/843,/0/default.jpg" alt="painting of people at a park on a sunny day" /> */}
```
Comment (Jayquintana) [inline src/CategoryCard/CategoryCard.tsx:12]: if were using this keep it if its not useful for later lets get rid of it

**Comment by mr-ryan12** [comment]
Thank you for this Stevie! Next steps will be to get the images displayed in the UI.

**Pull request merged**